packages/liveblocks-react/src/config.ts

```diff
@@ -19,4 +19,7 @@ export const config = {
 
   NOTIFICATION_SETTINGS_POLL_INTERVAL: 1 * MINUTES,
   NOTIFICATION_SETTINGS_MAX_STALE_TIME: 5 * SECONDS,
+
+  CHANNEL_NOTIFICATION_SETTINGS_INTERVAL: 1 * MINUTES,
+  CHANNEL_NOTIFICATION_SETTINGS_MAX_TIME: 5 * SECONDS,
```

Comment:
See my previous comment. If we keep the poller, I'd suggest to lower the polling interval for this resource to maybe every 10 minutes + 1 minutes respectively or so. The stale time can also be lowered a bit for this one I think.

For full context:

- Interval: 10 minutes
- Max stale time: 1 minute

Means:
- While the window remains focused and active, poll every 10 minutes
- If the window gets re-focused (user went to another app or browser tab, then comes back to this window) then: if the last time the poll happened was <= 1 minute ago, it won't do anything (next "normal" poll will automatically happen in < 10 mins)
- If the last time a poll happened was > minute ago, it will trigger a poll now. After that poll succeeds, the next "normal" poll will happen 10 mins later again.

Also, let's put "stale" in the config variable name at least:

```suggestion
  CHANNEL_NOTIFICATION_SETTINGS_INTERVAL: 10 * MINUTES,
  CHANNEL_NOTIFICATION_SETTINGS_MAX_STALE_TIME: 1 * MINUTES,
```

Reply:
See my [comment](https://github.com/liveblocks/liveblocks/pull/2093#discussion_r1865900369) above ☝🏻